tests/cache/test-vary-cache.php

```diff
@@ -669,9 +675,13 @@ public function get_test_data__parse_group_cookies() {
 					'iv' => '1231231231231234',
 				],
 				[
-					'vip-go-auth' => 'qSME2LdfVuNvZa0GfeUJ45/uHCu7Auqj3iesSL4CITteGfva/N0wQ5TCIzcyeBImeTf3On2P4f7EtQyviw2ooA==',
 				],
 				[
+					'HTTP_X_VIP_GO_AUTH' => 'vc-v1__design-group_--_yes---__dev-group_--_no'
+				],
+				[
+					'design-group' => 'yes' ,
+					'dev-group' => 'no'
```

Comment:
I think we still need a test case for the case where header doesn't exist and we fall to the cookie? (i.e. `if the header auth isn't set (in case of a logged-in user)`)